path_static_roles.go

```diff
@@ -280,14 +285,49 @@ func (b *backend) pathStaticRoleCreateUpdate(ctx context.Context, req *logical.R
 		role.StaticAccount.RotationPeriod = time.Duration(rotationPeriodSeconds) * time.Second
 	}
 
+	skipRotation := false
+	skipRotationRaw, ok := data.GetOk("skip_import_rotation")
+	if ok {
+		// if skip rotation was set, use it (or validation error on an update)
+		if !isCreate {
+			return logical.ErrorResponse("skip_import_rotation has no effect on updates"), nil
+		}
+		skipRotation = skipRotationRaw.(bool)
+	} else if isCreate {
+		// otherwise, go get it if this is a create request.
+		c, err := readConfig(ctx, req.Storage)
+		if err != nil {
+			return nil, errors.New("couldn't find configuration for this create operation's endpoint")
+		}
+		skipRotation = c.SkipStaticRoleImportRotation
+	}
+
 	// lvr represents the role's LastVaultRotation
 	lvr := role.StaticAccount.LastVaultRotation
 
-	// Only call setStaticAccountPassword if we're creating the role for the
-	// first time
+	// Only call setStaticAccountPassword if we're creating the role for the first time
 	var item *queue.Item
 	switch req.Operation {
 	case logical.CreateOperation:
+		// if we were asked to not rotate, just add the entry - this essentially becomes an update operation, except
+		// the item is new
+		if skipRotation {
+			entry, err := logical.StorageEntryJSON(staticRolePath+name, role)
+			if err != nil {
+				return nil, err
+			}
+			if err := req.Storage.Put(ctx, entry); err != nil {
+				return nil, err
+			}
+
+			// set the item
+			item = &queue.Item{
+				Key: name,
+			}
+			// synthetically set lvr to now, so that it gets queued correctly
+			lvr = time.Now()
+			break
```

Comment:
To future proof ourselves from accidentally skipping over additional logic that gets added in this `case` block, can we use `else` here?